### DISCUSSION TITLE
QFieldCloud import users from other project

### DISCUSSION THREAD

**Discussion by geomondego** [opengisch/QField#3014] [Ideas]
Hello OpengisTeam,  any future plans to import users from other project in QFieldCloud? 
Regards,
Nelson

**Comment by lucienicolier** [comment]
Hello @geomondego,

Is using QFieldCloud's teams can solve your issue? 

[QFieldCloud teams docs](https://docs.qfield.org/reference/qfieldcloud/concepts/?h=teams#organization-teams)

Best regards

**Reply by geomondego** [reply]
@lucienicolier i forget to look on documentation. I will check. Thanks
